### ISSUE TITLE
Using Table with rows prop prints findDOMNode warning in React strict mode in react-northstart

### ISSUE THREAD

**Issue by jschaf** [microsoft/fluentui#17328]
### Environment Information

- **Package version(s)**: (fill this out)

```
  "dependencies": {
    "@fluentui/react-northstar": "^0.52.1",
    "react": "^16.8.0",
    "react-dom": "^16.8.0",
    "scheduler": "^0.20.1"
  },
```

- **Browser and OS versions**: Chrome Version 88.0.4324.150 (Official Build) (64-bit)

### Please provide a reproduction of the bug in a codepen:

I wasn't able to reproduce in a codepen. Here's the smallest reproducible test case:

```ts
import React from 'react';
import ReactDOM from 'react-dom';
import {Table} from '@fluentui/react-northstar';

ReactDOM.render(
  <React.StrictMode>
    <Table rows={[{key: 1, items: [1]}]}/>
  </React.StrictMode>,
  document.getElementById('root'),
);
```

#### Actual behavior:

In strict mode, using table with rows prints a warning:

> Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of RefFindNode which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://fb.me/react-strict-mode-find-node
```
  in div (created by TableCell)
    in RefFindNode (created by Ref)
    in Ref (created by TableCell)
    in TableCell (created by TableRow)
    in div (created by TableRow)
    in RefFindNode (created by Ref)
    in Ref (created by TableRow)
    in TableRow (created by Table)
    in div (created by Table)
    in Table
    in StrictMode
```

#### Expected behavior:

There should be no warning.

### Priorities and help requested:

Are you willing to submit a PR to fix? No, I'm barely capable on the frontend.

Requested priority: Low

Products/sites affected: N/A

**Comment by gouttierre** [comment]
@jschaf - thanks for filing this issue. I see you couldn't provided a CodePen, let's see if the engineering team can continue with the information you have provided on the bug. 

@tomi-msft - If there enough information for you to assist here, would you be able to confirm if this is a regression, or if this behavior is an issue and there should be no warnings?

**Comment by jaz5hq** [comment]
https://codesandbox.io/s/fluent-ui-northstar-warning-demo-pzhm2

Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of FocusZone which is inside StrictMode.

Focus Zone has used "findDomNode" in this file "packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx" 
  setRef = (elem: React.ReactInstance): void => {
    // findDOMNode needed to get correct DOM ref with react-hot-loader, see https://github.com/gaearon/react-hot-loader/issues/964
    this._root.current = ReactDOM.findDOMNode(elem) as HTMLElement;
  };

**Comment by creage** [comment]
@gouttierre to me it fires on [this line](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-bindings/src/FocusZone/FocusTrapZone.tsx#L39)
```js
// @ts-ignore
  createRef = elem => {
    this._root.current = ReactDOM.findDOMNode(elem) as HTMLElement;
    // @ts-ignore
    this.windowRef.current = getWindow(this._root.current);
  };
```

**Comment by creage** [comment]
@gouttierre can we do something with this issue?

**Comment by khmakoto** [comment]
@creage if this is still relevant I can go ahead and reopen the issue for you. Is that the case here?

**Comment by creage** [comment]
@khmakoto yes it is. We still have that error in our console running `@fluentui/react-northstar` 0.60.0.

**Comment by khmakoto** [comment]
@creage I'm reopening this issue then, thanks for confirming!
